caas/kubernetes/provider/ingress.go

```diff
@@ -108,7 +153,7 @@ func (k *kubernetesClient) deleteIngress(name string, uid k8stypes.UID) error {
 }
 
 func (k *kubernetesClient) listIngressResources(labels map[string]string) ([]networkingv1beta1.Ingress, error) {
-	listOps := v1.ListOptions{
+	listOps := metav1.ListOptions{
 		LabelSelector: utils.LabelsToSelector(labels).String(),
 	}
 	ingList, err := k.client().NetworkingV1beta1().Ingresses(k.namespace).List(context.TODO(), listOps)
```

Comment:
I'm wondering why it's NetworkingV1beta1() and not NetworkingV1()?
This is an internal call only used by Juju etc, so no pod spec from the charm needs to be catered for